pkg/util/moving_sum.go

```diff
@@ -0,0 +1,95 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package util
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+type Bucket struct {
```

Comment:
question: is this part of the E2E test or an artifact from a different branch? ASC is a CODEOWNER of the `pkg/util` but I cannot find any call sites of this